source/src/cip/ciptypes.h

```diff
@@ -203,9 +201,7 @@ typedef struct {
  *
  */
 typedef struct {
-  EipUint8 path_size;
-  /**< Size of the Path in 16-bit words *//* TODO: Fix, should be UINT
-   (EIP_UINT16) */
+  EipUint16 path_size; /**< Size of the Path in 16-bit words */
   EipUint32 class_id; /**< Class ID of the linked object */
```

Comment:
As 32 bit class_ids are currently not valid (at least they are not mentioned to be allowed to have a 32 bit value in the logical segment), this field could be reduced to a 16 bit type.